Client/qtTeamTalk/chattextlist.h

```diff
@@ -49,6 +49,18 @@ class ChatTextList : public QListWidget, public ChatTextHistory
     QStringList allUrls(const QString &text) const;
     QString currentUrl(const QListWidgetItem* item) const;
     void limitText();
+    QAction *copyAct    = nullptr;

```

Comment:
Please use the 'm' prefix for member variables. I can squash the commit, so you don't need to fix commits.